.github/pull_request_template.md

```diff
@@ -1,10 +1,18 @@
-**Related Issue(s)**:  ...
+### Related Issues
+- fixes #issue-number
```

Comment:
Beware this might have undesired effects: sometimes we refer to an issue but we don't want to close it by merging the PR. Why not using comments like we do in the other sections?

```suggestion
- <!--- Link this PR to an existing issue, for example: fixes #1234, related to #2345 --> 
```

Reply:
I prefer the way it is right now actually. Leaving only a comment encourages people to simply list the issues: many don't know that adding `fixes` or `closes` has any effect, so they won't get the hint.

However having a PR that closes an issue is the most common case, so I'd rather default to that. Reviewers can always remove `fixes` from the original description if they decide it's not correct.

Reply:
@masci What do you think? I tend to agree with Sarah on this one.